src/fortran/fct.cpp

```diff
@@ -614,6 +621,17 @@ extern "C" {
         return 0;
     }
 
+    status_t th_getgibbs_rt_(const integer* n,
+                             doublereal* grt) {
```

Comment:
The `doublereal` typedef is deprecated. This should be just
```suggestion
    status_t th_getgibbs_rt_(const integer* n, double* grt) {
```

Reply:
I will also update existing routines to change `doublereal` to `double` and add those to the commit.